docs/admin/extensible-admission-controllers.md

```diff
@@ -105,7 +105,7 @@ webhooks:
 - name: <webhook name, e.g., pod-policy.example.io>
   rules:
   - apiGroups:
-    - ""
+    - "*"
```

Comment:
`""` is the legacy group. It's not a typo.

Reply:
I'm not sure now, I think ```""``` may actually include the quotes instead of resolving to be an empty string. I'm pretty sure that for some reason our yaml parser includes the quotes when you give it ```"*"``` but keeps just the asterisk when you give it ```'*'```, so here we might need to use ```''```. I'm going to double check this.